src/ioc_setup.ts

```diff
@@ -0,0 +1,107 @@
+import {deferUntilFactory} from "@nicolawealth/defer_until";
+
+export const iocFactory = () => {
```

Comment:
Should the filename by `ioc_factory` to follow the pattern of the filename matching the main export?